From 8815d340e508f0456bb56e9858c6cea3390a6143 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Tue, 15 Feb 2022 01:21:01 +0000
Subject: [PATCH] packit: remove unsupported -Dcryptolib=openssl option

Introduced later, so it breaks the build on v249-stable

Conflict:NA
Reference:https://github.com/systemd/systemd/commit/8815d340e508f0456bb56e9858c6cea3390a6143
---
 .packit.yml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.packit.yml b/.packit.yml
index 4545e30e08..ee47f8954e 100644
--- a/.packit.yml
+++ b/.packit.yml
@@ -31,6 +31,8 @@ actions:
     # [0] https://github.com/mesonbuild/meson/issues/7360
     # [1] https://github.com/systemd/systemd/pull/18908#issuecomment-792250110
     - 'sed -i "/^CONFIGURE_OPTS=(/a--werror" .packit_rpm/systemd.spec'
+    # cryptolib is supported from v250 and newer, remove it
+    - 'sed -i "/-Dcryptolib=openssl/d" .packit_rpm/systemd.spec'
 
 jobs:
 - job: copr_build
-- 
2.33.0

